bouffalo-hal/src/sdio.rs

```diff
@@ -321,22 +330,23 @@ pub struct Command(u16);
 
 /// CMD Type.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
-pub enum CmdType {
-    Other,
+pub enum SDHCmdType {
```

Comment:
```suggestion
pub enum CommandType {
```
I suggest using `CommandType` as the enum name, as the module name sdio already implies it pertains to an SDH command.